dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops

```diff
@@ -76,3 +76,15 @@ spi.extension.failed.to.construct.CWOWB1010E.useraction=Fix the error that is pr
 spi.extension.class.in.different.bundle.CWOWB1011E=CWOWB1011E: A {0} class registered through the CDI Extension SPI is in a different bundle than the CDIExtensionMetadata implementation.
 spi.extension.class.in.different.bundle.CWOWB1011E.explanation=The class is ignored because it was registered with the getExtensions() or getBeanClasses() method of a CDIExtensionMetadata implementation and is in a different bundle than the CDIExtensionMetadata implementation.
 spi.extension.class.in.different.bundle.CWOWB1011E.useraction=Move the affected class or create a new CDIExtensionMetadata implementation in the correct bundle.
+
+exception.creating.extensions.CWOWB1012E=CWOWB1012E: Exception creating extensions for the {0} archive. The error was: {1}
```

Comment:
suggested edit
Exception creating extensions for the {0} archive.
-->
CDI encountered an exception when it created extensions for the {0} archive.

If not "CDI", then substitute the entity that creates the extensions

Reply:
add a period after {1}

Reply:
I don't think we can safely put a period after `{1}`. The error message may itself be a translated string which could end in a period.